### DISCUSSION TITLE
Customizable padding for fitView method

### DISCUSSION THREAD

**Discussion by FelipeDecome** [xyflow/xyflow#4084] [New Features]
Hi guys,

Is it possible for you guys to update the method to also accept an array for the padding option?
My suggestion is to accept a number (for all sides) and an array to customize each side.
The array format could be this [top, right, bottom, left], following css order on margin/padding properties.

I am implementing a custom method myself, but I think this could be a good addition to the library.

**Comment by kongsgard** [comment]
This feature would greatly improve the utility of the fitView method.
How did you solve this in the meantime?
